operatorconfig/moduleconfig/authorization/v1.10.0/policies.yaml

```diff
@@ -263,3 +263,12 @@ data:
       claims == {}
       msg := sprintf("missing claims", [])
     }
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: redis-csm-cm
```

Comment:
LGTM, just needs to be in` v2.0.0-alpha` version instead of v1.10.0

Reply:
Done by mistake. Moved it to v2.0.0-alpha